plutus-core/untyped-plutus-core/test/Evaluation/Golden/iteAtHigherKind.type.golden

```diff
@@ -1 +1,3 @@
-(Left E015: Kind mismatch at () in type '(lam a_1 (type) (fun a_1 a_1))'. Expected kind '(type)' , found kind '(fun (type) (type))')
\ No newline at end of file
+(Left E015: Kind mismatch at () in type '(lam
```

Comment:
[There's](https://hackage.haskell.org/package/prettyprinter-1.7.0/docs/Prettyprinter-Internal.html#v:group), but it won't work for us, since we use a custom operator inserting `hardline`s everywhere. That stuff requires a separate refactoring. I changed pretty-printing of types back to what it was before this PR.